eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -220,6 +220,8 @@ stages:
                             -IssueNumber "$(Submitted.PullRequest.Number)" `
                             -Comment $prComment
                         displayName: Comment notification to PR
+                        # Failure to comment on the PR should not block release
+                        continueOnError: true
```

Comment:
This looks unrelated, but I believe I know why you need it now which is because the get-codeowners script was removed.

Reply:
This snuck in from some unrelated work